components/CreateOrganizationForm.js

```diff
@@ -1,289 +1,421 @@
+                      setAuthorization(checked);
+                    }}
+                  />
+                  <Flex justifyContent={['center', 'left']} my={4}>
+                    <StyledButton
+                      fontSize="13px"
+                      minWidth="148px"
+                      minHeight="36px"
+                      buttonStyle="primary"
+                      type="submit"
+                      loading={loading}
+                      onSubmit={handleSubmit}
+                      data-cy="cof-form-submit"
+                    >
+                      <FormattedMessage id="create.org.title" defaultMessage="Create Organization" />
+                    </StyledButton>
+                  </Flex>
 
-            <div className="tos">
-              <label>{intl.formatMessage(this.messages['tos.label'])}</label>
-              <div data-cy="tos">
-                <StyledCheckbox
-                  name="tos"
-                  id="oc-tos-checkbox"
-                  required
-                  onChange={({ target }) => this.handleChange('tos', target.checked)}
-                  label={
+                  <Box
+                    textAlign="left"
+                    fontSize="12px"
+                    minHeight="24px"
+                    color="black.500"
+                    fontWeight="normal"
+                    letterSpacing="0.5px"
+                  >
                     <FormattedMessage
-                      id="createcollective.tos.label"
-                      defaultMessage="I agree with the {toslink} of Open Collective."
-                      values={{
-                        toslink: (
-                          <StyledLink href="/tos" openInNewTab>
-                            <FormattedMessage id="tos" defaultMessage="terms of service" />
-                          </StyledLink>
-                        ),
-                      }}
+                      id="SignIn.legal"
+                      defaultMessage="By joining, you agree to our <TOSLink>Terms of Service</TOSLink> and <PrivacyPolicyLink>Privacy Policy</PrivacyPolicyLink>."
+                      values={I18nFormatters}
                     />
-                  }
-                />
-              </div>
-            </div>
-
-            <div className="actions">
-              <StyledButton
-                buttonStyle="primary"
-                type="submit"
-                onClick={this.handleSubmit}
-                disabled={loading || !this.state.modified}
-              >
-                {loading ? (
-                  <FormattedMessage id="loading" defaultMessage="loading" />
-                ) : (
-                  <FormattedMessage id="organization.create" defaultMessage="Create organization" />
-                )}
-              </StyledButton>
-            </div>
-          </div>
-        </div>
-      </div>
-    );
-  }
-}
+                  </Box>
+                  <Flex color="black.500" alignItems="center">
```

Comment:
`color` property would not apply to `Flex` - if you're trying to target the `P` items in the `Flex` apply the color property to them.